content/blog/2018-03-20-door.md

```diff
@@ -0,0 +1,179 @@
+---
+slug: "door"
+title: The Database of Odorant Responses
+package_version: 2.0.1
+authors:
+  - name: Daniel Münch
+    url: https://github.com/dahaniel
+date: 2018-03-20
+categories: blog
+topicid:
+tags:
+- r
+- community
+- software
+- review
+- onboarding
+- package
+- Drosophila
+- odor responses
+- physiology
+- neuroscience
+- data-only-pkg
+- visualization
+- modelling
+---
+
+# DoOR - The Database of Odorant Responses
+## Olfactory Coding
+Detecting volatile chemicals and encoding these into neuronal activity is a vital task for all animals that is performed by their olfactory sensory systems. While these olfactory systems vary vastly between species regarding their numerical complexity, they are amazingly similar in their general structure. The periphery of olfactory systems consists of different classes of olfactory sensory neurons (OSN). In mammals, OSNs are located in the nose, in insects, OSNs are located on the antenna. OSN classes are tuned to defined but overlapping sets of odors. Thus a single odor usually elicits differential responses across an ensemble of OSNs. This *ensemble code* is able to encode thousands of odors, even in comparably simple olfactory systems.
+
+To better understand the underlying logic of this ensemble code, one would ideally want to know the specific ensemble codes each and every chemical elicits across an olfactory system, the so-called *"olfactome"*. Obviously this is not feasible to tackle experimentally, yet many labs work with e.g. the olfactory system of *Drosophila* and publish studies containing responses of different odor-OSN combinations.
+
+DoOR takes these heterogeneous data measured in different labs, with different techniques in different metrics and maps/merges them into a common response space, creating a consensus data-set of all odor responses. By this DoOR provides access to the most-complete version of the *Drosophila* olfactome available.
```

Comment:
add comma after "By this"? otherwise I read right through it with wrong emphasis